streaming/src/test/scala/org/apache/gearpump/streaming/StreamingTestUtil.scala

```diff
@@ -37,4 +49,63 @@ object StreamingTestUtil {
     executorId += 1
     miniCluster.launchActor(props).asInstanceOf[TestActorRef[AppMaster]]
   }
+
+  def createEchoForTaskActor(taskClass: String, taskConf: UserConfig, system1: ActorSystem, system2: ActorSystem): (ActorRef, TestProbe) = {
+    import system1.dispatcher
```

Comment:
I think it is difficult to test TaskActor because it is very stick to the transport layer so that we could not verify the output of TaskActor easily. I will try to find a solution to that.